consume_messages.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk'
+
+# Bring environment variables in ./config/.env into scope.
+# This is probably only used in development.
+require 'dotenv'
+Dotenv.load(File.join('.', 'config', '.env'))
+
+# Create a hash that contains the configuarable variables
+require 'erb'
+require 'yaml'
+path_to_settings = File.join(__dir__, "config", "settings.yml")
+settings = YAML.load(ERB.new(File.read(path_to_settings)).result)
+
+# Configure SQS Client
+credentials = Aws::Credentials.new(settings['aws_key'], settings['aws_secret'])
+sqs_client  = Aws::SQS::Client.new(region: 'us-east-1', credentials: credentials)
+
+response = sqs_client.receive_message(queue_url: settings['sqs_queue_url'], attribute_names: ['All'])
+
```

Comment:
Here's where we retrieve and iterate over messages from SQS.
There's no guarantee about how many messages come back from `sqs_client.receive_message()`
but I've read online not to expect more than a few unless you have LOTS of messages in your queue.